packages/airnode-node/src/workers/local-handlers.ts

```diff
@@ -12,6 +13,11 @@ function loadConfig() {
 
 export async function startCoordinator(): Promise<WorkerResponse> {
   const config = loadConfig();
+  // Set the Airnode wallet private key into the environment when running locally
+  // eslint-disable-next-line functional/immutable-data
+  process.env['AIRNODE_WALLET_PRIVATE_KEY'] = ethers.Wallet.fromMnemonic(
+    config.nodeSettings.airnodeWalletMnemonic
+  ).privateKey;
```

Comment:
Hmm, this works for the other handlers just because the `startCoordinator` is called first and the env variables are shared. That's kinda meh.  What about creating a function and calling it for each handler the same way we do with `loadConfig`?

Reply:
>  That's kinda meh.

:smile: Indeed it is, added.